### PR TITLE
Added explanation to relationships in documentation

### LINKED ISSUE #397: Creation of VRF Documentation
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### APIC version and APIC Platform

* V x.x.x and on-prem/cloud-aws/cloud-azure/all.

### Affected Resource(s)

* aci_XXXXX

### Terraform Configuration Files

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
```

### Steps to Reproduce

Notice in aci_vrf, we create a VRF, not a ctx and though there are obscure references to the ctx identity, we don’t really explain the relationship.
https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/vrf
 
But in aci_bridge_domain, we reference relation_fv_rs_ctx to reference the VRF we create. There is really nothing on the aci_bridge_domain  reference page to tell us that the relation_fv_rs_ctx is for the VRF reference.
https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/bridge_domain
 
Would suggest that:
We show the relation_fv_rs_ctx in the example usage
We ensure that for the argument reference for relation_fv_rs_ctx description we make it clear that this is referencing the VRF for the bridge domain.
 
Figured this out from this site:
https://terra-lab.ciscolive.com/lab/pod20/part3/tenant
 

1. `terraform apply`

### References

* #0000

### PR DESCRIPTION
**Pull request by anvitha-jain**
fixes #397

### CODE REVIEW

File: website/docs/r/vrf.html.markdown [hunk 1]
```diff
@@ -10,6 +10,15 @@ description: |-
...
 Manages ACI VRF
 
+## API Information ##
+
+* `Class` - fvCtx
+* `Distinguished Named` - uni/tn-{name}/ctx-{name}
+
+## GUI Information ##
+
+* `Location` - Tenant -> Networkinh -> VRFs
```
Comment (lhercot) [inline website/docs/r/vrf.html.markdown:20]: Networkinh -> Networking

File: website/docs/r/vrf.html.markdown [hunk 2]
```diff
@@ -49,6 +58,8 @@ resource "aci_vrf" "foovrf" {
 * `relation_fv_rs_ctx_to_ext_route_tag_pol` - (Optional) Relation to class l3extRouteTagPol. Cardinality - N_TO_ONE. Type - String.
 * `relation_fv_rs_ctx_to_bgp_ctx_af_pol` - (Optional) Relation to class bgpCtxAfPol. Cardinality - N_TO_M. Type - [Set of Map].
 
+Note: In the APIC GUI,a VRF (fvCtx) is also called a "Context"or "PrivateNetwork."
```
Comment (lhercot) [inline website/docs/r/vrf.html.markdown:61]: I would use might/was called as we have almost changed it everywhere now. (Or should have)

**Review by lhercot — CHANGES_REQUESTED** [review]

File: examples/bridge_domain/bd.tf [hunk 3]
```diff
@@ -23,16 +23,16 @@ resource "aci_bridge_domain" "demobd" {
...
   vmac                           = "not-applicable"
-  relation_fv_rs_bd_to_profile   = aci_rest.rest_rt_ctrl_profile.id # Relation to rtctrlProfile class. Cardinality - N_TO_ONE
-  relation_fv_rs_bd_to_relay_p   = aci_rest.rest_dhcp_RelayP.id     # Relation to dhcpRelayP class. Cardinality - N_TO_ONE
-  relation_fv_rs_abd_pol_mon_pol = aci_rest.rest_mon_epg_pol.id     # Relation to monEPGPol class. Cardinality - N_TO_ONE
-  relation_fv_rs_bd_flood_to     = [aci_filter.bd_flood_filter.id]  # Relation to vzFilter class. Cardinality - N_TO_M
-  relation_fv_rs_bd_to_fhs       = aci_rest.rest_fhs_bd_pol.id     # Relation to fhsBDPol class. Cardinality - N_TO_ONE.
+  relation_fv_rs_bd_to_profile   = aci_rest.rest_rt_ctrl_profile.id # Relation to L3Outs Route Map For Import and Export Route Control
+  relation_fv_rs_bd_to_relay_p   = aci_rest.rest_dhcp_RelayP.id     # Relation to DHCP Relay policy
+  relation_fv_rs_abd_pol_mon_pol = aci_rest.rest_mon_epg_pol.id     # Relation to Monitors policy
+  relation_fv_rs_bd_flood_to     = [aci_filter.bd_flood_filter.id]  # Relation to Filters
```
[the remark this answers is not in the document]
Comment (anvitha-jain) [reply examples/bridge_domain/bd.tf:29]: yes

File: examples/bridge_domain/bd.tf [hunk 4]
```diff
@@ -11,7 +11,7 @@ resource "aci_bridge_domain" "demobd" {
   optimize_wan_bandwidth         = "no"
   arp_flood                      = "no"
   ep_clear                       = "no"
-  ep_move_detect_mode            = "0"
+  // ep_move_detect_mode            = "0"
```
[the remark this answers is not in the document]
Comment (anvitha-jain) [reply examples/bridge_domain/bd.tf]: ep_move_detect_mode  should be = "garp" and if this parameter is not required the remove the parameter (assigning to "0" will not work)

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**